pages/index.tsx

```diff
@@ -1,14 +1,229 @@
-import Dashboard from "@components/features/Dashboard";
 import LoggedOutOnlyRoute from "@components/shared/layouts/LoggedOutOnlyRoute";
 import type { NextPage } from "next";
-import Head from "next/head";
-import Image from "next/image";
+import { motion, useInView } from "framer-motion";
 import styles from "../styles/Home.module.css";
+import { Button } from "@components/shared/elements/Button";
+import Link from "next/link";
+import Image from "next/image";
+import { ResponseTestIcon } from "@components/features/Dashboard/Applications/ResponseTest";
+import { InteractiveQuizIcon } from "@components/features/Dashboard/Applications/InteractiveQuiz";
+import { VideoConferenceIcon } from "@components/features/Dashboard/Applications/VideoConference";
+import { OdiLogo } from "@components/shared/svg/OdiLogo";
+
+const buttonStyles: React.CSSProperties = {
+  margin: 0,
+  fontSize: "1.5rem",
+  fontWeight: "bold",
+  padding: 0,
+  minWidth: 200,
+  height: 60,
+};
+
+const Roller: React.FC<{ words: string[] }> = ({ words }) => {
+  return (
```

Comment:
Hello, rashad. Thanks for implementing interactive components like this roller right here! I think it'll give a better impresskon for our website. 😁

Reply:
Thank you Alex! This was all I could come up with upon our first release. Hopefully in a later update there will be more showcased in the home page: D

Reply:
I believe so rashad! We can add more improvements on later updates. 😄